src/attached_probe.cpp

```diff
@@ -694,16 +693,13 @@ void AttachedProbe::load_prog()
 
     // Use orig_name for program name so we get proper name for
     // wildcard probes, replace wildcards with '.'
```

Comment:
Can you adjust this comment accordingly to the change? There's no more `orig_name` identifier.

Reply:
Doesn't orig_name refer to probe_.orig_name in this context ?

Reply:
Ah, that's probably correct, sorry about the confusion.